stdlib/collections/__init__.pyi

```diff
@@ -206,20 +206,9 @@ class deque(MutableSequence[_T], Generic[_T]):
     def __len__(self) -> int: ...
     def __iter__(self) -> Iterator[_T]: ...
     def __str__(self) -> str: ...
-    # These methods of deque don't really take slices, but we need to
```

Comment:
Can you keep a version of this comment to explain why we need the ignores?

Reply:
Sure

Reply:
Done!